exporter/templates_test.go

```diff
@@ -354,6 +351,72 @@ func TestDiagnosticStudyPerformedTemplate(t *testing.T) {
 	assertXPath(t, xrn, "//entry/observation/effectiveTime/high", map[string]string{"value": "201505200810+0000"}, nil)
 }
 
+func TestPatientCharacteristicClinicalTrialParticipantTemplate(t *testing.T) {
+	qrdaOid := "2.16.840.1.113883.10.20.24.3.51"
+	dataCriteriaName := "patient_characteristic_clinical_trial_participant"
+	entryName := "patient_characteristic_clinical_trial_participant"
+
+	ei := generateDataForTemplate(dataCriteriaName, entryName, &models.Encounter{})
+
+	xrn := xmlRootNodeForQrdaOidWithData(qrdaOid, ei)
+
+	assertXPath(t, xrn, "//entry/observation", map[string]string{"classCode": "OBS", "moodCode": "EVN"}, nil)
+	assertXPath(t, xrn, "//entry/observation/templateId[@root='2.16.840.1.113883.10.20.24.3.51']", nil, nil)
+	assertXPath(t, xrn, "//entry/observation/code", map[string]string{"code": "ASSERTION", "codeSystem": "2.16.840.1.113883.5.4"}, nil)
+	assertXPath(t, xrn, "//entry/observation/effectiveTime/low", map[string]string{"value": "201507130830+0000"}, nil)
+
```

Comment:
Can remove extra empty line at end of function. Ditto for the other new test functions here.